pkg/apis/monitoring/v1/podmonitor_types.go

```diff
@@ -123,6 +123,11 @@ type PodMonitorSpec struct {
 	//
 	// +optional
 	LabelValueLengthLimit *uint64 `json:"labelValueLengthLimit,omitempty"`
+
+	// Optional NativeHistogramConfig.
+	// +optional
+	NativeHistogramConfig `json:",inline"`
```

Comment:
the field being inlined, I think that the comment isn't used by the doc/API generation. Same for all the other types.
```suggestion
	NativeHistogramConfig `json:",inline"`
```